freqtrade/freqai/freqai_interface.py

```diff
@@ -253,25 +257,46 @@ def start_backtesting(
             dataframe_train = dk.slice_dataframe(tr_train, dataframe)
             dataframe_backtest = dk.slice_dataframe(tr_backtest, dataframe)
 
+            if dk.backtest_live_models and len(dataframe_backtest) == 0:
+                tr_backtest_startts_str = datetime.fromtimestamp(
+                                                tr_backtest.startts,
+                                                tz=timezone.utc).strftime(DATETIME_PRINT_FORMAT)
+                tr_backtest_stopts_str = datetime.fromtimestamp(
+                                                tr_backtest.stopts,
+                                                tz=timezone.utc).strftime(DATETIME_PRINT_FORMAT)
+                logger.info(
+                    f"No data found for pair {pair} "
+                    f" from {tr_backtest_startts_str} "
+                    f"to {tr_backtest_stopts_str}. "
```

Comment:
in general i'd try to use as few lines as possible (which means - avoid unnecessary line wrapping ...)